src/appkit.cs

```diff
@@ -23671,4 +23676,10 @@ interface NSOpenGLLayer
 		void Draw (NSOpenGLContext context, NSOpenGLPixelFormat pixelFormat, double t, ref CVTimeStamp ts);
 	}
 
+	[Protocol (IsInformal=true)]
```

Comment:
@spouliot @rolfbjarne Is it correct to add:

	[BaseType (typeof(NSObject))]

here? After testing, I realized that XS does not offer this as an autocompletion on INSToolTipOwner unless you have this.